sample/OSCON17/tracks

```diff
@@ -0,0 +1,133 @@
+[{
+		"id": 1,
+		"name": "Open source: From consumer to contributor",
+		"color": "#3DC8C3",
+		"sessions": [{
+				"id": 9,
+				"title": "How can I contibute? A guide to making your first open source contribution"
+			},
+			{
+				"id": 2,
+				"title": "Step 1: Punch a tree"
+			},
+			{
+				"id": 3,
+				"title": "Why choose open infrastructure?"
+			},
+			{
+				"id": 17,
+				"title": "Cloud native Java"
+			}
+		],
+		"track_image_url": "",
+		"font_color": "white"
```

Comment:
are we planning to use them in the app?
well its better to use hex if we tend to use them in future.